docs/layers.md

```diff
@@ -1468,7 +1468,7 @@ The `hgv` property indicates general truck heavy goods vehicle truck access, val
 
 **Roads `hgv_restriction` values:**
 
-For `hgv_restriction` property indicates general truck heavy goods vehicle truck access restrictions, values include: `weight` (kg), `height` (cm), `length` (cm), `width` (cm), `wpa` (weight per axle, in kg), `kpra` (king pin to rear axle leght, in cm), `hazmat`, `other` and `multiple` if more than one.
+For `hgv_restriction` property indicates general truck heavy goods vehicle truck access restrictions, values include: `weight` (metric tonnes), `height` (metres), `length` (metres), `width` (metres), `wpa` (weight per axle, in metric tonnes), `kpra` (king pin to rear axle leght, in metric tonnes), `hazmat` (true if restricted, otherwise omitted), `other` and `multiple` if more than one.
```

Comment:
It's fine to change the units on this, but let's coordinate with @musculman.

Reply:
Yup! Good idea.

I don't consider this to be a _change_ per se, since it was previously documented as cm & kg for `hgv_restriction` and metres & tonnes for `hgv_restriction_shield_text`. I followed the documentation for `hgv_restriction_shield_text`, and the diff above merely updates the conflicting documentation for `hgv_restriction` to match.